app/views/clients/cas_readiness/_manual.haml

```diff
@@ -49,6 +49,10 @@
     .col-lg-8.col-xl-6
       = render 'vispdat', f: f
 
```

Comment:
For future reference any changes also need to be added to the "_file.haml" file so that they'll appear when either of these are chosen.
![screen shot 2018-12-31 at 9 59 38 am](https://user-images.githubusercontent.com/1346876/50562573-d6f47d00-0ce2-11e9-8a34-02e4fada741a.png)